include/openssl/core_names.h

```diff
@@ -184,6 +184,9 @@ extern "C" {
 #define OSSL_PKEY_PARAM_RSA_FACTOR      "rsa-factor"
 #define OSSL_PKEY_PARAM_RSA_EXPONENT    "rsa-exponent"
 #define OSSL_PKEY_PARAM_RSA_COEFFICIENT "rsa-coefficient"
+/* Key generation parameters */
+#define OSSL_PKEY_PARAM_RSA_BITS        "bits"
```

Comment:
why have these ones lost the "rsa-" prefix ?

Reply:
Why should it have one? Are the numbers of rsa bits something different than, say, the amount of dsa bits?

Reply:
So you really mean:
#define OSSL_PKEY_PARAM_BITS        "bits"
#define OSSL_PKEY_PARAM_RSA_BITS OSSL_PKEY_PARAM_BITS

Reply:
Good point